paper/README.qmd

```diff
@@ -1,149 +1,131 @@
 
-our algorithm can be thought of as an extension of buffer and orientation analysis
+This section details the ANIME algorithm, from its conceptual foundation and geometric matching procedures to its methods for attribute integration and its implementation.
 
-- location / proximity measured as a distance buffer (of sorts)
-- orientation (angle)
+## Algorithm Overview
 
-- where we differ is in the use of length
-- most existing algorithms perform matching between the source and target based on a measure of length of each LineString. Common are average euclidean distance, hausdorff, and frechet distance.
-- each of these evaluates the linestrings in their entirety 
-- the algorithms then use the spatial criteria to perform a binary classification of either a match or not
-- we do
+A key feature of the ANIME algorithm is its support for `m:n` correspondence, a departure from many existing algorithms that focus on 1:1 matching. Each feature in the target FeatureCollection can be matched to zero, one, or multiple features in the source FeatureCollection. The approach accommodates all relationship cardinalities, "including one-to-null, null-to-one, one-to-one, one-to-many, many-to-one, and many-to many" (Chehreghan & Abbaspour, 2017). Instead of providing a binary classification, the algorithm identifies potential matches and quantifies their correspondence, allowing for subsequent pruning if desired.
 
-## Algorithm overview 
+The proposed algorithm aims to match elements of two sets of LineStrings that are topologically similar and to estimate the amount of shared length between each pair of matched LineStrings. The result of the matching algorithm is a data structure (specifically, a B-Tree map) that contains the matched pairs and their shared lengths, which can be readily converted to a sparse matrix representation.
 
+| Variable/Parameter           | Symbol    | Description                                                                    |
+| ---------------------------- | --------- | ------------------------------------------------------------------------------ |
+| Source features              | $A$       | The input `FeatureCollection` from which features are being matched.           |
+| Target features              | $B$       | The input `FeatureCollection` to which features are being matched.             |
+| Distance Tolerance           | $DT$      | The maximum distance for a line segment in B to search for matches in A.       |
+| Angle Tolerance              | $AT$      | The maximum angle difference between two line segments to be considered parallel. |
+| Shared Length                | $SL_{ij}$ | The length of overlap between a matched source feature $i$ and target feature $j$. |
+| Source-weighted proportion | -         | The proportion of a source feature that is matched to a target feature.        |
+| Target-weighted proportion | -         | The proportion of a target feature that is matched to a source feature.        |
```

Comment:
these are not actually part of the core data structure but are returned for ergonomic reasons so that users of the client library want to do their own thing